xocto/fields/postgres/ranges.py

```diff
@@ -93,7 +93,20 @@ def _upper_to_inclusive(self, value: pg_ranges.DateRange) -> datetime.date:
         return value.upper - datetime.timedelta(days=1)
 
 
-class FiniteDateTimeRangeField(pg_fields.DateTimeRangeField):
+class _ConvertibleDateTimeRangeMixin:
+    def __init__(self, *args: Any, as_localtime: bool = True, **kwargs: Any):
+        super().__init__(*args, **kwargs)
+        self._as_localtime = as_localtime
+
+    def convert(self, value: datetime.datetime) -> datetime.datetime:
+        if self._as_localtime:
+            return localtime.as_localtime(value)
+        return localtime.as_utc(value)
```

Comment:
❓ I'm not comfortable with this naming:
* `_ConvertibleDateTimeRangeMixin`
* `as_localtime`
* `convert`

 Are you thinking of something better ?

Reply:
`_Localiser`
`localise`

?

Reply:
Much better thanks!